core/templates/pages/exploration-editor-page/services/populate-rule-content-ids.service.spec.ts

```diff
@@ -31,11 +31,11 @@ describe('Populate Rule Content Ids Service', () => {
   it('should populate null content ids on save', () => {
     let rule = {
       type: 'Equals',
-      inputTypes: {x: 'TranslatableSetOfNormalizedString'},
-      inputs: {x: {
+      inputTypes: { x: 'TranslatableSetOfNormalizedString' },
+      inputs: { x: {
         contentId: null,
         normalizedStrSet: []
-      }}
+      } }
```

Comment:
```suggestion
      } 
    }
```

Reply:
@DubeySandeep I have done this change at this place, but there are many other places too with this style. Shouldn't we create a seperate linter for this?